pyomo/contrib/doe/fim_doe.py

```diff
@@ -205,26 +208,38 @@ def __generate_flatten_timeset(self, all_info, flatten_measure_name,name_and_ind
         self.flatten_measure_timeset = flatten_measure_timeset
 
     def __model_measure_name(self):
-        '''Return pyomo string name 
+        '''Return pyomo string name
```

Comment:
This function does not return anything. Instead, it updates self.measurement_names_ef and self.measurement_names_kaug.